.github/workflows/sync-testsite-branch.yml

```diff
@@ -0,0 +1,18 @@
+name: Sync Main to Storybook Site Branch
+
+on:
+  push:
+    branches:
+      - main
+jobs:
+  sync-branches:
```

Comment:
is it possible to combine this workflow with the storybook site one so we don't need two?

Reply:
I kinda like it separate..but sure

Reply:
maybe just a callable workflow that's reused with a variable passed in if we want to keep them as separate?

Reply:
eh, combining it is fine